docs/docs/0.3.2/policies/README.md

```diff
@@ -425,6 +429,60 @@ In particular, `prometheus.io/*` annotations are part of configuration used by `
 If you're using a different way to install `Prometheus` on `kubernetes`, those annotations might not have the desired effect.
 :::
 
+Although it's easy and works without any extra configuration, those annotations supports only one address to scrape.
+If you also have an application next to Kuma DP which exposes metrics through `prometheus.io/*` annotations,
+you have to use the other way to expose metrics for both the application and Kuma DP.
+
+##### Kuma Prometheus SD
+
+You can deploy `kuma-prometheus-sd` container next to you Prometheus instance just like in Universal setup.
+
+First, add a volume to your Prometheus deployment to which `kuma-prometheus-sd` will write a file with the list of the dataplanes and from which `Prometheus` will read the list.
```

Comment:
We could provide automation for the first 3 steps in the form of `k8s` mutating web hook:
* we could create a custom hook (similarly the one we use to inject `kuma-dp`)
* or, we could provide instructions how to use [OPA](https://www.openpolicyagent.org/docs/v0.12.2/kubernetes-admission-control/) for that

With such automation in place, the instructions will look like:
1. Add an extra `annotation`/`label` to your Prometheus `Pods`
2. Add an extra configuration snippet to your `prometheus.yml` 

Another advantage of mutating web hook is that it will be automatically upgrading the version of `kuma-prometheus-sd` once a newer version of `kuma-injector` is installed

Reply:
Created trello card for this.